CHANGELOG.md

```diff
@@ -1,6 +1,102 @@
+* Add region to boto3 sqs/s3 clients. See [#266](https://github.com/tilezen/tilequeue/pull/266).
+* Resolve new flake8 errors. See [#267](https://github.com/tilezen/tilequeue/pull/267).
+* Log coordinates as integers. See [#268](https://github.com/tilezen/tilequeue/pull/268).
+* Fix some RAWR-related TODOs. See [#269](https://github.com/tilezen/tilequeue/pull/269).
+* Dockerize Tilequeue. See [#270](https://github.com/tilezen/tilequeue/pull/270).
+* Use underscore as the separator instead of dash. See [#271](https://github.com/tilezen/tilequeue/pull/271).
+* Handle the case where the queue returns no msgs. See [#272](https://github.com/tilezen/tilequeue/pull/272).
+* Add support for local storage of RAWR tiles.. See [#273](https://github.com/tilezen/tilequeue/pull/273).
+* Emit more detailed timing and stats. See [#274](https://github.com/tilezen/tilequeue/pull/274).
+* Log when rawr processing starts/stops. See [#275](https://github.com/tilezen/tilequeue/pull/275).
+* Correct all calls to create rawr enqueuer's. See [#276](https://github.com/tilezen/tilequeue/pull/276).
+* Add configuration comment about metatile size 2. See [#278](https://github.com/tilezen/tilequeue/pull/278).
+* Updates for tilequeue processing on dev. See [#279](https://github.com/tilezen/tilequeue/pull/279).
+* Remove gzip rawr formatter. See [#280](https://github.com/tilezen/tilequeue/pull/280).
+* Various RAWR tile fixes. See [#281](https://github.com/tilezen/tilequeue/pull/281).
+* Add a running section to README.. See [#282](https://github.com/tilezen/tilequeue/pull/282).
+* Update all packages to the latest versions.. See [#288](https://github.com/tilezen/tilequeue/pull/288).
+* Simplify coord -> queue msg handle mappings. See [#290](https://github.com/tilezen/tilequeue/pull/290).
+* Updates for sqs message visibility handling. See [#293](https://github.com/tilezen/tilequeue/pull/293).
+* Add support for configured nominal zoom to single tile processing.. See [#294](https://github.com/tilezen/tilequeue/pull/294).
+* Add command to enqueue randomly sampled pyramids. See [#295](https://github.com/tilezen/tilequeue/pull/295).
+* Log additional details on msg ack errors. See [#296](https://github.com/tilezen/tilequeue/pull/296).
+* Simplify `_fetch_and_output` function. See [#298](https://github.com/tilezen/tilequeue/pull/298).
+* Update tilequeue proc stats. See [#299](https://github.com/tilezen/tilequeue/pull/299).
+* Enqueue random samples to rawr queue. See [#300](https://github.com/tilezen/tilequeue/pull/300).
+* Add extra data tables. See [#301](https://github.com/tilezen/tilequeue/pull/301).
+* Add additional rawr intersector implementations. See [#302](https://github.com/tilezen/tilequeue/pull/302).
+* Update rawr intersect config parsing. See [#303](https://github.com/tilezen/tilequeue/pull/303).
+* Same precision for metatiles. See [#304](https://github.com/tilezen/tilequeue/pull/304).
```

Comment:
Is this the MVT precision changes? If so please indicate the precision has improved to be consistent between 256, 512, snd 1024 metatiles in the MVT format, something something.